mily/tests/test_imports.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+
+
+@pytest.mark.parametrize("widget", [
+    "MTableItemDelegate", "MTableInterfaceView", "MTableInterfaceWidget",
+    "MFunctionTableInterfaceWidget",
+    "MText", "MISpin", "MFSpin", "MComboBox", "MCheckBox", "MSelector", "MDateTime",
+    "MetaDataEntry", "vstacked_label", "hstacked_label",
+])
+def test_import_widgets(widget):
+    code = f"from mily.widgets import {widget}"
```

Comment:
Can we use `importlib` instead of `exec`? https://docs.python.org/3.8/library/importlib.html#importlib.import_module

Reply:
Done